app/presenters/formats/place_presenter.rb

```diff
@@ -0,0 +1,42 @@
+module Formats
+  class PlacePresenter < EditionFormatPresenter
+  private
+
+    GOVSPEAK_FIELDS = %i(
+      introduction
+      more_information
+      need_to_know
+    ).freeze
+
+    def schema_name
+      'place'
+    end
+
+    def details
+      details = { external_related_links: external_related_links }
+
+      details = details.merge(govspeak_fields)
+
+      if @edition.place_type
+        details = details.merge(place_type: @edition.place_type)
```

Comment:
there's an attribute reader for the `@edition` variable, so we can lose the snail.